post_processing/define_tc_quicklook_profiles.jl

```diff
@@ -89,8 +85,12 @@ function plot_tc_profiles(folder; hdf5_filename, main_branch_data_path)
         plot!(p16, parent(D.env_HQTcov)[:], zc; label = "$data_source")
     end
 
+    PR_filename = joinpath(folder, hdf5_filename)
     add_to_plots!(PR_filename; data_source = "PR")
-    add_to_plots!(main_filename; data_source = "main")
+    if ispath(main_branch_data_path)
+        main_filename = joinpath(main_branch_data_path, hdf5_filename)
+        add_to_plots!(main_filename; data_source = "main")
+    end
```

Comment:
```suggestion
    else
        @warn "No path found for comparison against main"
    end
```

Reply:
I added the warning [here](https://github.com/CliMA/ClimaAtmos.jl/blob/ba696500f562eb49ff60f61719c10cec4b6ad5b7/regression_tests/self_reference_or_path.jl#L89), in the same function where I moved the comparison `@info`.